src/js/components/TextArea/__tests__/TextArea-test.tsx

```diff
@@ -183,6 +190,9 @@ describe('TextArea', () => {
       fireEvent.change(input, {
         target: { value: 'Test' },
       });
+      if (!(input instanceof HTMLTextAreaElement)) {
```

Comment:
Could we change line 189 to `const input = getByPlaceholderText('item') as HTMLInputElement;` and then remove this check?